test/MagFieldLFT_test.jl

```diff
@@ -727,6 +727,171 @@ function test_print_composition_ErIII()
+    r_Ni = [0.000,   0.000,   0.000]                      # atom 33
+    r_NH = [0.511,  -2.518,  -0.002]                      # atom 49  NH
+    r_CH1 = [1.053,   1.540,   3.541]                     # atom 23  CH'
+    r_CH2 = [-0.961,  -1.048,  -3.741]                    # atom 32  CH
+    r_alpha1_alpha2prime_1 = [-1.500,  -3.452,   1.130]   # atom 44  alpha1
+    r_alpha1_alpha2prime_2 = [0.430,  -3.104,   2.402]    # atom 45  alpha2'
+    R_NH                   = r_Ni - r_NH
+    R_CH1                  = r_Ni - r_CH1
+    R_CH2                  = r_Ni - r_CH2
+    R_alpha1_alpha2prime_1 = r_Ni - r_alpha1_alpha2prime_1
+    R_alpha1_alpha2prime_2 = r_Ni - r_alpha1_alpha2prime_2
+    R = [R_NH, R_CH1, R_CH2, R_alpha1_alpha2prime_1, R_alpha1_alpha2prime_2] / bohrinangstrom
+
+    param = read_AILFT_params_ORCA("NiSAL_HDPT.out", "CASSCF")
+    T = 298. 
+    B0_MHz = 10. 
+    B0 = B0_MHz/42.577478518/2.35051756758e5
+    S = 1.0
+
+    shift_ord4 = MagFieldLFT.calc_shifts_KurlandMcGarvey_ord4(param, R, T, B0)
+    KMcG_shifts = MagFieldLFT.calc_shifts_KurlandMcGarvey(param, R, T)
+    Br_shifts = MagFieldLFT.calc_shifts_KurlandMcGarvey_Br(param, R, T, B0, S)
+
+    return norm(KMcG_shifts - Br_shifts) < 1.0e-4  && norm(KMcG_shifts - shift_ord4) < 1.0e-4
+end
+
+#at very high temperature the difference in field dependent effects simulation should decrease
+#due to the population of all the levels of the ground multiplet  
+function test_KurlandMcGarvey_ord4_Br_temperature()
+    bohrinangstrom = 0.529177210903
+    # atom counting starting from 1 (total number of atoms is 49, NH proton is the last one)
+    r_Ni = [0.000,   0.000,   0.000]                      # atom 33
+    r_NH = [0.511,  -2.518,  -0.002]                      # atom 49  NH
+    r_CH1 = [1.053,   1.540,   3.541]                     # atom 23  CH'
+    r_CH2 = [-0.961,  -1.048,  -3.741]                    # atom 32  CH
+    r_alpha1_alpha2prime_1 = [-1.500,  -3.452,   1.130]   # atom 44  alpha1
+    r_alpha1_alpha2prime_2 = [0.430,  -3.104,   2.402]    # atom 45  alpha2'
+    R_NH                   = r_Ni - r_NH
+    R_CH1                  = r_Ni - r_CH1
+    R_CH2                  = r_Ni - r_CH2
+    R_alpha1_alpha2prime_1 = r_Ni - r_alpha1_alpha2prime_1
+    R_alpha1_alpha2prime_2 = r_Ni - r_alpha1_alpha2prime_2
+    R = [R_NH, R_CH1, R_CH2, R_alpha1_alpha2prime_1, R_alpha1_alpha2prime_2] / bohrinangstrom
+
+    param = read_AILFT_params_ORCA("NiSAL_HDPT.out", "CASSCF")
+    T = 400. 
+    B0_MHz = 400. 
+    B0 = B0_MHz/42.577478518/2.35051756758e5
+    S = 1.0
+
+    shift_ord4 = MagFieldLFT.calc_shifts_KurlandMcGarvey_ord4(param, R, T, B0)
+    Br_shifts = MagFieldLFT.calc_shifts_KurlandMcGarvey_Br(param, R, T, B0, S)
+
+    # println(shift_ord4, Br_shifts)
+    # println(norm((shift_ord4 - Br_shifts) .* 100 ./ shift_ord4))
+
+    return norm((shift_ord4 - Br_shifts) .* 100 ./ shift_ord4) < 5.0e-2
+end
+
+
+function fielddepshift_cube()
```

Comment:
This function (fielddepshift_cube) seems to be in the wrong place. Can you delete it (and possibly preserve it in a different branch)?